consensus/consensus.go

```diff
@@ -136,6 +142,9 @@ type Engine interface {
 	// GetLinkExternalBlocks links every block to the correct previous block
 	GetLinkExternalBlocks(chain ChainHeaderReader, header *types.Header, logging bool) ([]*types.ExternalBlock, error)
 
+	// PreviousCoincidentOnPath searches the path for a block of specified order in the specified slice
```

Comment:
Not just a block of order in the slice, right? You're looking for the block in a specific context/path within that slice, right?

e.g. look for the last region block in the zone path/context.
e.g. look for the last zone block (in my slice!) in the region path/context.

Or am I confused?

Reply:
Just saw the implementation, and think it has a more fitting description. Might just copy paste it from there:
https://github.com/spruce-solutions/go-quai/blob/35453ba180754e4b7da1798dc79272fecd68f8e8/consensus/blake3/consensus.go#L706

Reply:
Expand the link. There are 3 more lines of docs that I was suggesting should go here. No biggie

Reply:
```
// PreviousCoincidentOnPath searches the path for a block of specified order in the specified slice
//     *slice - The zone location which defines the slice in which we are validating
//     *order - The order of the conincidence that is desired
//     *path - Search among ancestors of this path in the specified slice
```